admin/settings.php

```diff
@@ -126,23 +126,30 @@ public function init_settings() {
 				if(!isset($field["register"]) || $field["register"]) {
 					register_setting($this->option_group, $this->prefix.$field["name"]);
 				}
-				
+
 			}
 		}
 	}
-	
+
 	/**
 	 * Add Settings submenu and page
 	 *
 	 * @since  0.10
 	 * @return void
 	 */
 	public function add_settings_menu() {
+
+		$cap = RUA_App::CAPABILITY;
+		if ( ! is_multisite() && is_super_admin() && ! current_user_can( $cap ) ) {
+			// Replicate multisite super admin behavior for single installations.
+			$cap = 'manage_options';
```

Comment:
I think with this approach we can scrap the is_level_admin and the additional checks, but I can just cherrypick from this PR instead of undoing all that.

However, a problem with using a new capability and not adding it to a role (such as adminsitrator), is that it then can't be found anywhere (or at least I can't see how), and that means that the capability cannot be granted/denied by this plugin.

Also, if super admins are the only ones allowed to manage restrictions, what will that mean for site admins in multisite networks?

Sorry for this taking so long.